models/visualization/actions_diffs_count.sql

```diff
@@ -140,7 +140,7 @@ internal_links_out_count_diff,
 top_3_keywords_diff,
 top_10_keywords_diff
 FROM {{ ref('actions_diffs') }}
-WHERE content_action not like 'Rising%' and content_action not like 'Below%'
+WHERE content_trajectory not like 'Rising%' and content_action not like 'Below%'
```

Comment:
Let's revisit this filter - don't think we need this `WHERE` parameter (probably just the `AND`s).